Assets/Scripts/CharacterMovement.cs

```diff
@@ -0,0 +1,193 @@
+﻿using UnityEngine;
+using System.Collections;
+
+[RequireComponent(typeof(Animator))]
+[RequireComponent(typeof(CharacterController))]
+public class CharacterMovement : MonoBehaviour
+{
+
+    Animator animator;
+    CharacterController characterController;
+    
+    //Serialized classes
+    [System.Serializable]
+    public class AnimationSettings
+    {
+        public string verticalVelocityFloat = "Forward";
```

Comment:
Place a comment here to show that these are the 'fields' where values are stored in the animator. Naming a variable as float will be confusing as you've typed it a string.